config/samples/dataplane_v1beta1_openstackdataplanenodeset_ceph_hci.yaml

```diff
@@ -4,18 +4,10 @@ metadata:
   name: openstack-edpm-hci
   namespace: openstack
 spec:
-  env:
-  - name: ANSIBLE_CALLBACKS_ENABLED
-    value: profile_tasks
-  - name: ANSIBLE_FORCE_COLOR
-    value: "True"
-  - name: ANSIBLE_ENABLE_TASK_DEBUGGER
-    value: "True"
```

Comment:
This is going to fail actually. I need to address this in ci-framework and install_yamls before it will work here.

Tracking via:
https://issues.redhat.com/browse/OSPRH-2284

Reply:
I might just leave it in for now and then address the env: params in a seperate PR to reduce complexity and the number of Depends-On required.